modules/gathering-operator-logs.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting-operator-issues.adoc
+
+[id="gathering-operator-logs_{context}"]
+= Gathering Operator logs
+
+Gather detailed diagnostic information from Operator Pod logs.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* Your API service is still functional.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* You have the fully-qualified domain names of the control plane, or master, machines.
+
+.Procedure
+
+. List Operator Pods running in the Operator's namespace, plus Pod status, restarts and age:
++
+----
+$ oc get pod -n <operator_namespace>
+----
+
+. Review logs for an Operator Pod:
++
+----
+$ oc logs --tail 3 -n <operator_namespace>
+----
++
+If an Operator Pod has multiple containers, the preceding command will produce an error that includes the name of each container. Query logs from an individual container:
++
+----
+$ oc logs --tail 3 -n <operator_namespace> -c <container_name> <operator_pod_name>
+----
+
+. If the API is not functional, review Operator Pod and container logs on each master node by using SSH instead. Replace `<master-node>.<cluster_name>.<sub_domain>.<domain>` with appropriate values:
+.. List Pods on each master node:
++
+----
+$ ssh core@<master-node>.<cluster_name>.<sub_domain>.<domain> sudo crictl pods
+----
++
+.. For any Operator Pods not showing `Ready` status, inspect Pod status in detail. Replace `<operator_pod_id>` with the Operator Pod's ID listed in the output of the preceding command:
```

Comment:
I think this sentence is missing a few "the"'s

Reply:
Resolved.